### PR TITLE
Historical EPI data fix for piloting districts

### PR DESCRIPTION
**Pull request by gcapalbo**
Adds a command to recompute aggregates at the country level for historical epi data.

@czue

### CODE REVIEW

File: logistics_project/apps/malawi/warehouse/runner.py [hunk 1]
```diff
@@ -497,10 +497,15 @@ def update_consumption_times(since):
...
+    This function only works against base level data.
     """
     # any consumption value that was touched potentially needs to have its
     # time_needing_data updated
-    consumptions_to_update = CalculatedConsumption.objects.filter(update_date__gte=since)
+    consumptions_to_update = CalculatedConsumption.objects.filter(
+        Q(product__type__base_level=BaseLevel.HSA, supply_point__type__code=SupplyPointCodes.HSA) |
+        Q(product__type__base_level=BaseLevel.FACILITY, supply_point__type__code=SupplyPointCodes.FACILITY),
+        update_date__gte=since
+    )
```
Comment (gcapalbo) [inline logistics_project/apps/malawi/warehouse/runner.py]: @czue it seemed to me like this should only operate on base level data, is that right?

with the previous behavior even if it were to catch and update aggregated data, the aggregated data would normally be fixed automatically when the aggregates run in the same warehouse run.  but it causes a problem when we fix data historically like in this pr in which case the aggregates would not run again for past months.

Comment (czue) [inline logistics_project/apps/malawi/warehouse/runner.py]: hey, that sounds right to me. based on where this is called in the normal warehouse workflow I think maybe it works fine because the only possible `CalculatedConsumption`s that would be updated since the warehouse started are the base level ones, yeah? (assuming nothing else is touching them concurrently). Either way being explicit about it and not assuming anything about the context around which it is called makes sense to me.

Comment (czue) [inline logistics_project/apps/malawi/warehouse/runner.py]: is this relevant to this PR or just something you noticed? I may be missing something but I don't see how this would get called by `update_aggregated_data`.

Comment (gcapalbo) [inline logistics_project/apps/malawi/warehouse/runner.py]: yeah this only gets run during the warehouse runner, it won't be called by the management command introduced in this pr.

i think i misunderstood what this was doing - i thought it was pulling records updated since the end of the last warehouse run, but it's only pulling records updated since the beginning of the current run. so the problem i was envisioning won't be a problem and i reverted this commit.

it does seem that it would only ever pull base level data given the point at which this is run in the runner, but including that big or statement in there on such a big table would just slow down the query.
